website/src/hooks/useDailySummaryData.ts

```diff
@@ -0,0 +1,19 @@
+import useApiCall from "@/utils/Hook";
+import { DailySummarydata, Workloads } from "@/types";
+
+export default function useDailySummaryData(workloads: Workloads[]) {
+
+  const workloadsQuery = workloads.join('&workloads=');
+
+  const {
+    data: dataDailySummary,
+    isLoading: isLoadingDailySummary,
+    error: errorDailySummary,
+  } = useApiCall<DailySummarydata>(`${import.meta.env.VITE_API_URL}daily/summary?workloads=${workloadsQuery}`);
```

Comment:
We should support the case where we do not send any workload and want to rely on the default mechanism of the API (just `GET` on `/api/daily/summary`, without any argument).